airbase/pom.xml

```diff
@@ -178,6 +178,10 @@
         <!-- See http://stackoverflow.com/questions/1012402/maven2-property-that-indicates-the-parent-directory    -->
         <air.main.basedir>${project.basedir}</air.main.basedir>
 
+        <!-- ${argLine} is used by Jacoco to store commands to enable its agent, and then used in Surefire configuration.
+             When Jacoco is not enabled, it will stay empty, otherwise it will be overwritten. -->
```

Comment:
Yes, I did. I know it works because in Trino Jacoco complains about too-new bytecode version (I'm running Java 18) :)

```
$ mvnd -pl :trino-tests test -Dtest=TestVerifyTrinoTestsTestSetup -Dair.check.skip-jacoco=false

...

[INFO] --- jacoco-maven-plugin:0.8.7:prepare-agent (default) @ trino-tests ---
[INFO] argLine set to -javaagent:/Users/kudivuhadi/.m2/repository/org/jacoco/org.jacoco.agent/0.8.7/org.jacoco.agent-0.8.7-runtime.jar=destfile=/private/var/tmp/trino/testing/trino-tests/target/jacoco.exec

...

[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ trino-tests ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running io.trino.tests.TestVerifyTrinoTestsTestSetup
[WARN] [stderr] 2022-06-22T09:13:14.490-0500 INFO LogTestDurationListener enabled: false
[WARN] [stderr] 2022-06-22T09:13:14.517-0500 INFO ProgressLoggingListener is disabled!
[WARN] [stderr] java.lang.instrument.IllegalClassFormatException: Error while instrumenting sun/util/resources/cldr/provider/CLDRLocaleDataMetaInfo.
...
[WARN] [stderr] Caused by: java.lang.IllegalArgumentException: Unsupported class file major version 62
...
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.427 s - in io.trino.tests.TestVerifyTrinoTestsTestSetup
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] Segment walltime 3 s, segment projects service time 3 s, effective/maximum degree of concurrency 1,00/15
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.893 s (Wall Clock)
[INFO] Finished at: 2022-06-22T16:13:14+02:00
[INFO] ------------------------------------------------------------------------
```